functions.php

```diff
@@ -485,4 +485,12 @@ function nightingale_check_author( $display_name ) {
     }
     return $display_name;
 }
-add_filter( 'the_author', 'nightingale_check_author', PHP_INT_MAX, 1 );
\ No newline at end of file
+add_filter( 'the_author', 'nightingale_check_author', PHP_INT_MAX, 1 );
+
+/**
+ * For security prevent password reset error message providing info on user emails
+ */
+function nightingale_no_login_hints ( $error ) {
+	return 'If your email has been found in our database you will receive a reset link';
```

Comment:
can this be translatable please?
return _e( 'If your email has been found in our database you will receive a reset link', 'nightingale);

Reply:
Good point. Done.